pkg/codegen/nodejs/test.go

```diff
@@ -63,20 +63,15 @@ func TypeCheck(t *testing.T, path string, _ codegen.StringSet, linkLocal bool) {
 }
 
 func typeCheckGeneratedPackage(t *testing.T, pwd string, linkLocal bool) {
-	// NOTE: previous attempt used npm. It may be more popular and
-	// better target than yarn, however our build uses yarn in
-	// other places at the moment, and yarn does not run into the
-	// ${VERSION} problem; use yarn for now.
-
-	test.RunCommand(t, "yarn_install", pwd, "yarn", "install")
+	test.RunCommand(t, "npm_install", pwd, "npm", "install")
 	if linkLocal {
 		test.RunCommand(t, "yarn_link", pwd, "yarn", "link", "@pulumi/pulumi")
```

Comment:
@tgummerer should this `yarn link` also be migratd to `npm link`?

Reply:
I don't think we can, since we only use `yarn link` in the Makefile to create the development package.  I don't think the two are interchangeable, so I left `yarn link` here in place for now.  It works because they are both working on the same `node_modules` folder.

Longer term we were also planning to remove some of the `yarn link` stuff as well, so I think it's okay to keep this oddity here for now.